pkg/storegateway/postings_codec.go

```diff
@@ -97,6 +105,141 @@ func diffVarintSnappyDecode(input []byte) (index.Postings, error) {
 	return newDiffVarintPostings(raw), nil
 }
 
+// isDiffVarintSnappyEncodedPostings returns true, if input looks like it has been encoded by diff+varint+snappy+matchers codec.
+func isDiffVarintSnappyWithmatchersEncodedPostings(input []byte) bool {
```

Comment:
`s/matchers/Matchers/` here and everywhere please.

Reply:
this is so long that I never read it again after mistyping it

Reply:
`func isDiffVarintSnappyWithSomethingOhDearWhyAreYouEvenReadingThisPostings(input []byte) bool {`